ddtrace/internal/runtime/tag_collectors.py

```diff
@@ -35,13 +37,20 @@ class PlatformTagCollector(RuntimeTagCollector):
       - For Jython this is 'Jython'.
     - lang_version:
       - eg. '2.7.10'
+    - lang:
+      - e.g. 'Python'
+    - tracer_version:
+      - e.g. '0.29.0'
     """
-    required_modules = ['platform']
+    required_modules = ('platform', 'ddtrace')
 
     def collect_fn(self, keys):
         platform = self.modules.get('platform')
+        ddtrace = self.modules.get('ddtrace')
         tags = [
+            (LANG, 'Python'),
```

Comment:
```suggestion
            (LANG, 'python'),
```

it is going to get lowercased anyways